SYCL/ESIMD/api/functional/ctors/ctor_fill_accuracy_fp_extra.cpp

```diff
@@ -0,0 +1,61 @@
+//==------- ctor_fill_accuracy_fp_extra.cpp  - DPC++ ESIMD on-device test --==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu, level_zero
+// XREQUIRES: gpu
+// TODO gpu and level_zero in REQUIRES due to only this platforms supported yet.
+// The current "REQUIRES" should be replaced with "gpu" only as mentioned in
+// "XREQUIRES".
+// UNSUPPORTED: cuda, hip
+// XRUN: %clangxx -fsycl %s -fsycl-device-code-split=per_kernel -o %t.out
+// XRUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: false
+// XFAIL: *
+// TODO The simd can't be constructed with sycl::half data type. The issue was
+// created (https://github.com/intel/llvm/issues/5077) and the test must be
+// enabled when it is resolved.
+//
+// The test verifies that simd fill constructor has no precision differences.
+// The test do the following actions:
+//  - call simd with predefined base and step values
+//  - bitwise comparing that output[0] value is equal to base value and
+//    output[i] is equal to output[i -1] + step_value
+
+#include "ctor_fill.hpp"
+
+using namespace sycl::ext::intel::experimental::esimd;
+using namespace esimd_test::api::functional::ctors;
+using namespace esimd_test::api::functional;
```

Comment:
```suggestion
using namespace esimd_test::api::functional;
```
What do you think?

Reply:
this comment was applied [here](https://github.com/intel/llvm-test-suite/pull/598/commits/04d6bab9c3fb63aeef04dbeea45f802d05edce82#diff-7bc755870d95f59f1370a04cf56a312662d85c6d91a6e6176d4537f89e4e304cL30) and in other files